.github/workflows/pr-title-check.yml

```
@@ -0,0 +1,27 @@
+  pull_request:
+    types: [opened, edited, synchronize]
+    branches: [main, develop]
+
+jobs:
+  pr-title-check:
+    name: 지라 티켓 ID 형식 확인
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 📋 PR 제목 형식 검증
+        run: |
```

actionlint (1.7.10)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)